src/api/todos.tsx

```diff
@@ -0,0 +1,27 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const deleteTodo = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const createTodo = ({
+  title,
+  completed,
+  userId,
+}: Omit<Todo, 'id'>) => {
+  return client.post<Todo>('/todos/', { title, userId, completed });
+};
```

Comment:
```suggestion
export const createTodo = (newTodo: Omit<Todo, 'id'>) => {
  return client.post<Todo>('/todos/', newTodo);
};
```
In this case I think you don't need destructing